integer/src/lib.rs

```diff
@@ -723,6 +764,46 @@ fn test_lcm_overflow() {
 }
 
 #[test]
+fn test_binomial_coeff() {
```

Comment:
Perhaps this needs to use one of `binomial`'s inputs that tread near overflow conditions?